tests/helpers/tags.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  Interface_OrganizationTagUser,
+  OrganizationTagUser,
+  TagUser,
+} from "../../src/models";
+import { nanoid } from "nanoid";
+import {
+  createTestUserAndOrganization,
+  testUserType,
+  testOrganizationType,
+  createTestUser,
+} from "./userAndOrg";
+
+export type testUserTagType = Interface_OrganizationTagUser | null;
```

Comment:
`type` and `interface` in typescript are named using PascalCase. Fix all the places you've used snakeCase.

Reply:
@xoldyckk But it would require refactoring of all the test files as the same convention has been used across all test objects types, as explained in the previous comment.

Reply:
/Should I just rename the `testUserTagType` to `TestUserTagType`?

Reply:
Yeah rename it. I noticed the usage of snakeCase for naming typescript types, that's a big typescript anti-pattern.

Reply:
Got it.